### PR TITLE
JAMES-3441 Disable task consumption configuration

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

**Comment by chibenwa** [comment]
test this please

File: dockerfiles/run/guice/cassandra-rabbitmq-ldap/destination/conf/rabbitmq.properties [hunk 1]
```diff
@@ -39,4 +39,9 @@ mailqueue.view.updateBrowseStartPace=1000
 # Choose to disable it when the mail queue size is getting too big
 # Note that this is as well a temporary workaround until we get 'getSize' method better optimized
 # Optional, default false
-mailqueue.size.metricsEnabled=false
\ No newline at end of file
+mailqueue.size.metricsEnabled=false
+
+# Whether to enabled task consumption on this node. Tasks are WebAdmin triggered long running jobs.
```
Comment (Arsnael) [inline dockerfiles/run/guice/cassandra-rabbitmq-ldap/destination/conf/rabbitmq.properties:44]: s/enabled/enable

File: dockerfiles/run/guice/cassandra-rabbitmq/destination/conf/rabbitmq.properties [hunk 2]
```diff
@@ -39,4 +39,9 @@ mailqueue.view.updateBrowseStartPace=1000
 # Choose to disable it when the mail queue size is getting too big
 # Note that this is as well a temporary workaround until we get 'getSize' method better optimized
 # Optional, default false
-mailqueue.size.metricsEnabled=false
\ No newline at end of file
+mailqueue.size.metricsEnabled=false
+
+# Whether to enabled task consumption on this node. Tasks are WebAdmin triggered long running jobs.
```
Comment (Arsnael) [inline dockerfiles/run/guice/cassandra-rabbitmq/destination/conf/rabbitmq.properties:44]: idem

File: docs/modules/servers/pages/distributed/configure/rabbitmq.adoc [hunk 3]
```diff
@@ -66,3 +66,19 @@ Tips for choosing metricsEnabled are explained in
...
+
+Tasks are WebAdmin triggered long running jobs. RabbitMQ is used to organise their execution in a work queue,
+with an exclusive consumer.
+
+.rabbitmq.properties content
+|===
+| Property name | explanation
+
+| task.consumption.enabled
+| Whether to enabled task consumption on this node.
```
Comment (Arsnael) [inline docs/modules/servers/pages/distributed/configure/rabbitmq.adoc:80]: idem

File: src/site/xdoc/server/config-rabbitmq.xml [hunk 4]
```diff
@@ -107,6 +107,20 @@
...
   </section>
 
+  <section name="RabbitMQ Tasks Configuration">
+      <p>Tasks are WebAdmin triggered long running jobs. RabbitMQ is used to organise their execution in a work queue,
+          with an exclusive consumer.
+      </p>
+      <dl>
+          <dt><strong>task.consumption.enabled</strong></dt>
+          <dd>
+              Whether to enabled task consumption on this node.
```
Comment (Arsnael) [inline src/site/xdoc/server/config-rabbitmq.xml:117]: idem

**Review by Arsnael — APPROVED** [review]

**Review by Arsnael — APPROVED** [review]

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Review by LanKhuat — APPROVED** [review]

**Pull request merged**